WordPress/Classes/Models/WPAccount+Lookup.swift

```diff
@@ -26,7 +26,7 @@ public extension WPAccount {
     ///
     @objc
     var hasBlogs: Bool {
-        return !blogs.isEmpty
+        return blogs?.isEmpty == false
```

Comment:
(nit) Is there any way to make `blogs` in `WPAccount` nonnull for convenience?

Reply:
At runtime, it probably won't be null. Core Data _should_ return an empty list if there is no `Blog` instance associated with this account. However, considering an "Optional" property in the relationship configuration, I feel it is safer to declare it nullable.